include/SampleTrack.h

```diff
@@ -58,6 +58,7 @@ class SampleTCO : public TrackContentObject
 		return m_sampleBuffer;
 	}
 
+	void setStartTimeOffset( MidiTime startTime );
```

Comment:
@jasp00 excellent point, yes this will be useful for other track types.  Piano Roll, BBEdtior and Automation patterns can eventually benefit from this as well. 👍